### PR TITLE
WAN Assurance plugin 3.7.4 relese notes

### PR DESCRIPTION
**Pull request by agrawalkaushik**
- Captures content and bug fixes for Jun, Jul and Aug pushes

### CODE REVIEW

File: docs/release_notes_wan_assurance_plugin_3.7.md [hunk 1]
```diff
@@ -2,6 +2,70 @@
...
+
+  _**Resolution:**_ The interface name is made part of the key for tracking DHCP server events to allow both events to be sent.
+
+- **WAN-1933 The URL filtering status is not getting updated in MIST for conductor managed devices**
+
+  _**Resolution:**_ Added support for sending the URL filtering status for routers running SSR version < 6.1.0.
+
+- **WAN-1944 The cloud-intel agent does not honor the configured log level**
+
+  _**Resolution:**_ The log-level configuration setting is correctly propagated to teh cloud-intel-agent.
```
Comment (BenMatase) [inline docs/release_notes_wan_assurance_plugin_3.7.md:50]: `teh` -> `the`

Comment (BenMatase) [reply docs/release_notes_wan_assurance_plugin_3.7.md:50]: inconsistent reference to cia. In title it is `cloud-intel agent` and here it is `cloud-intel-agent`

File: docs/release_notes_wan_assurance_plugin_3.7.md [hunk 2]
```diff
@@ -2,6 +2,70 @@
...
+
+  _**Resolution:**_ Better handling of the negative value conditions when leases have been reclaimed by the DHCP server which resulted in the bad values to be reported.
+
+- **WAN-1753 The cloud-intel-agent service is running even when WAN assurance is not enabled**
+
+  _**Resolution:**_ The service will no longer be launched on system startup automatically unless the WAN assurance plugin is enabled.
+
+- **WAN-1817 Releasing conductor managed SSR from MIST wipes config**
+
+  _**Resolution:**_ The system will now skip factory reset for a conductor managed greenfield device .
```
Comment (BenMatase) [inline docs/release_notes_wan_assurance_plugin_3.7.md:38]: space before period at the end

File: sidebars.js [hunk 3]
```diff
@@ -275,7 +275,7 @@ module.exports = {
       "bcp_tenants",
     ],
     "CLI and Element Reference": [
-      "cli_reference",
+      // "cli_reference",
```
Comment (BenMatase) [inline sidebars.js:278]: intentional?

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]
Just Ben's suggestions, otherwise it looks good.

File: sidebars.js (hunk 3 above)
Comment (agrawalkaushik) [reply sidebars.js:278]: oops - not intentional at all - will remove.

**Review by gregschrock — APPROVED** [review]

**Review by Chr1st0ph3rTurn3r — APPROVED** [review]

**Review by BenMatase — DISMISSED** [review]

File: docs/release_notes_wan_assurance_plugin_3.7.md [hunk 4]
```diff
@@ -2,6 +2,70 @@
 title: WAN Assurance Plugin 3.7 Release Notes
 sidebar_label: '3.7'
 ---
+## Release 3.7.4
+**Release Date:** Aug 20, 2023
```
Comment (MichaelBaj) [inline docs/release_notes_wan_assurance_plugin_3.7.md:6]: This should be September 20

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**